src/navsat_transform.cpp

```diff
@@ -557,6 +586,18 @@ namespace RobotLocalization
 
   void NavSatTransform::odomCallback(const nav_msgs::OdometryConstPtr& msg)
   {
+    if (world_frame_param_loaded_ && (world_frame_id_.compare(msg->header.frame_id) != 0))
```

Comment:
`compare` will obviously work, but I think this would be more readable as `world_frame_id_ != msg->header.frame_id`. Same below with the `base_link_frame_id_`.